### PR TITLE
Chanatpakorn/joh 58 sprint 4 download and share

### PR DESCRIPTION
**Pull request by ChanatpakornS**
- add Download and Share to Horo Page

petHoro-The High Priestess

minor bug?? : help me!

### CODE REVIEW

**Review by boomchanotai — CHANGES_REQUESTED** [review]
/game รูปไม่ขึ้น

JohnJud result

**Comment by ChanatpakornS** [comment]
- Implementing download and share to `/game` and `/horo`
- deployment failed จากไรวะ
- canvas so hard, i'm using plain canvas for now
- Art later

**Comment by ChanatpakornS** [comment]
- Changing to use picture assets instead
- Change some wrong result in Horo
- Rename utils download and share
- create two utils becuz if there any decision changes

File: .env.example
Comment (boomchanotai) [inline .env.example:1]: เอามันกลับมา

File: src/utils/Results.ts
Comment (boomchanotai) [inline src/utils/Results.ts:1]: rename to `gameResult`

File: src/utils/horo.ts
Comment (boomchanotai) [inline src/utils/horo.ts:1]: rename to `horoResult`

File: src/utils/shareResult.ts
Comment (boomchanotai) [inline src/utils/shareResult.ts:1]: อันนี้ใช้หรอ

**Review by boomchanotai — CHANGES_REQUESTED** [review]

File: src/utils/shareResult.ts
Comment (ChanatpakornS) [reply src/utils/shareResult.ts:1]: deleted in the new commit

**Comment by ChanatpakornS** [comment]
- refactor filename
- sorry for .env.example deletion
- deleted canvas-txt lib and shareResult since it's no use
-

**Review by boomchanotai — APPROVED** [review]

**Pull request merged**